language-plutus-core/examples/Language/PlutusCore/Examples/Data/InterList.hs

```diff
@@ -7,8 +7,11 @@ module Language.PlutusCore.Examples.Data.InterList
     , getBuiltinFoldrInterList
     ) where
 
-import           Language.PlutusCore
 import           Language.PlutusCore.MkPlc
+import           Language.PlutusCore.Name
+import           Language.PlutusCore.Quote
+import           Language.PlutusCore.Renamer
+import           Language.PlutusCore.Type
```

Comment:
I do not have anything against this change, but I wonder why you made it.

Reply:
I was doing something earlier that gave me an import cycle, but I don't need it now. OTOH I think importing the "everything" module inside `language-plutus-core` is somewhat bad practice anyway.